src/components/SaveGameForm.js

```diff
@@ -1,23 +1,26 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 import {useState} from 'react'
+import saveLocally from '../lib/saveLocally'
+import loadlocally from '../lib/loadLocally'
 import Button from './Button'
 
-
 SaveGameForm.propTypes = {
     onSubmit: PropTypes.func.isRequired
   }
 
 export default function SaveGameForm({onSubmit}) {
    
-    const [formInput, setFormInput] = useState({
+    const [formInput, setFormInput] = useState(loadlocally('formInput') ?? {
         location: '',
         date: '',
         players: '',
         winner: '',
         shots:'',
     })
 
+    saveLocally('formInput', formInput)
```

Comment:
My thought on this was to save the input values also during typing. So that, incase the page is rerendering, the user does not have to type in the values again. Does it make sense?